README.md

```diff
@@ -1,17 +1,28 @@
-# Socket Component
+# react/socket
```

Comment:
Tbh, i don't like using the lowercase composer handle as the readme title :(
My personal preference would be just `Socket`. I'd even prefer the old title over `react/socket`.